### PR TITLE
Relocate input module section

### PR DESCRIPTION
**Pull request by emiglietta**
I moved the optional section on how to configure the input modules towards the end, bc it usually disrupts the flow of the workshops and gets participants confused.

Also added a Pipeline Summary figure for guiding participants.

### CODE REVIEW

File: BeginnerSegmentation/segmentation_start.cppipe [hunk 1]
```diff
@@ -5,7 +5,7 @@ GitHash:
...
+Images:[module_num:1|svn_version:'Unknown'|variable_revision_number:2|show_window:False|notes:['WELCOME TO THE BEGGINNER SEGMENTATION TUTORIAL!! :D', '', 'To begin creating your project, use the Images module to compile a list of files and/or folders that you want to analyze. You can also specify a set of rules to include only the desired files in your selected folders.', 'Drag and drop your IMAGES (you can drag and drop the entire "images_Illum-corrected" folder) into the panel just below this one.', '', '<-Drag and drop your "segmentation_start.cppipe" files into the left panel.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```
Comment (bethac07) [inline BeginnerSegmentation/segmentation_start.cppipe:8]: Beginner

**Comment by bethac07** [comment]
@emiglietta Can I ask you to please also pull #125, and to update at least the English zip artifact? Since all the downloads point to the zip artifacts, people won't get the updated version unless that's done as well. Up to you if once #125 is pulled (and then #126 is auto-created by Transifex), you also want to do updated Spanish and Portuguese PDFs + zips but let's at least make sure the versions stay in-line. Thanks!

**Comment by emiglietta** [comment]
Hi! I'm on it! thank you for the raminder! I made sure to update the English version of the zip before the workshop but was waiting for the Transifex notification that the outo translation was done. 
I will add the translation to my TODO list for later and leave it with the automatic version for now :)

**Comment by emiglietta** [comment]
Done!

**Pull request merged**